src/plugins/navigation/public/top_nav_menu/top_nav_control_item.tsx

```diff
@@ -52,7 +52,10 @@ export function TopNavControlItem(props: TopNavControlData) {
       <EuiText className="descriptionHeaderControl" size="s">
         {props.description}
         {links?.map((linkProps) => (
-          <TopNavControlItem {...linkProps} />
+          <>
```

Comment:
Do we need to add a Fragment here?

Reply:
nah. The empty tag is just for the comment.